apps/antalmanac/src/components/Map/Map.tsx

```diff
@@ -217,7 +217,7 @@ export default function CourseMap() {
         <Box sx={{ width: '100%', display: 'flex', flexDirection: 'column', flexGrow: 1, height: '100%' }}>
             <MapContainer ref={map} center={[33.6459, -117.842717]} zoom={16} style={{ height: '100%' }}>
                 {/* Menu floats above the map. */}
-                <Paper sx={{ zIndex: 400, position: 'relative', my: 2, mx: 6.942, marginX: '15%', marginY: 8 }}>
+                <Paper sx={{ zIndex: 400, position: 'relative', my: 2, mx: 8 }}>
```

Comment:
LGTM to me but the x margin seems a tad low

![chrome-capture-2023-9-5](https://github.com/icssc/AntAlmanac/assets/100006999/6506841e-5e3d-43e2-a638-4cf35f114265)

Reply:
![image](https://github.com/icssc/AntAlmanac/assets/66761366/0edf8880-23ef-489d-9331-ef34de627565)